src/test/java/com/google/api/generator/gapic/dummy/BUILD.bazel

```diff
@@ -21,3 +23,39 @@ filegroup(
         "@junit_junit//jar",
     ],
 ) for test_name in TESTS]
+
+# JUnit runner binary, this is used to generate test output for updating goldens files.
+java_binary(
+    name = "junit_runner_binary",
+    srcs = [
+        ":dummy_files",
+        "//src/test/java/com/google/api/generator/test/framework:framework_files",
+    ],
+    data = ["//src/test/java/com/google/api/generator/gapic/dummy/goldens:goldens_files"],
```

Comment:
Since I moved this to root `BUILD.bazel` the `goldens_files` data stays the same, while I changed the goldens files attr in `java_test` to `glob(["goldens/*.golden"])`